docs/references/debugging-sdk-tools.md

```diff
@@ -0,0 +1,82 @@
+# Debugging SDK Tools
+Now that you've built your first SDK Tool, you want to confirm that it's properly installed and that the code you've written is being executed. This guide will show you places to check for correctness and how to verify that your plugin is being correctly built and executed. It is recommended to follow these steps when behavior is not acting as expected, and to always rebuild using the `create-yxi` command to iterate and ensure that changes are being made to your plugin code.  
+
+
+## Table of Contents
+1. After `create-yxi`
+2. After installing `.yxi`
+3. After running in Designer
+4. Running the plugin at Command Line
+5. Simple ways to print/check values
+
+
+## 1. After `create-yxi`
+After creating the yxi, there will be a new folder in your workspace called `build`. Navigate to `~/your_tool_workspace/build/yxi/workspace_name.yxi` and rename the `.yxi` to a `.zip` archive. 
+
+Inside the archive, navigate to `<TOOL_NAME>/site-packages/ayx_plugins/<tool_name>.py`. When opening this file, you should expect to see the exact same code as the plugin code in your tool workspace (`~/your_tool_workspace/backend/ayx_plugins/<tool_name>.py`). If you make some changes to your code in your workspace, all updates should also propogate to this file that's in the `.yxi` that you build. This step is used to confirm that the `.yxi` is being built with the correct code and dependencies. You can verify that the other plugin dependencies are also being correctly built by simply searching for that dependency in the `site-packages` folder of the archive. 
```

Comment:
TW review:

After you create the YXI, there is a new folder in your workspace called `build`. Navigate to `~/your_tool_workspace/build/yxi/workspace_name.yxi` and rename the `.yxi` to a `.zip` archive. 

Inside the archive, navigate to `<TOOL_NAME>/site-packages/ayx_plugins/<tool_name>.py`. When you open this file, you should see the exact same code as the plugin code in your tool workspace (`~/your_tool_workspace/backend/ayx_plugins/<tool_name>.py`). If you make some changes to your code in your workspace, all updates should also propagate to this file that's in the `.yxi` that you build. This step is used to confirm that the `.yxi` is built with the correct code and dependencies. You can also verify that the other plugin dependencies are also correctly built. To do this, search for that dependency in the `site-packages` folder of the archive.

Reply:
`You can also verify that the other plugin dependencies are also correctly built. To do this, search for that dependency in the site-packages folder of the archive.`

don't like how this flows. `also` used twice in first sentence. also breaking the sentence into 2 different sentences makes the first sentence feel like it's not saying anything 

can we rephrase?

Maybe switch the order and mention the site-packages stuff first and segue it into checking any other dependencies?

Reply:
@ayx-orion thanks for the note, how about this:

Inside the archive, navigate to <TOOL_NAME>/site-packages/ayx_plugins/<tool_name>.py. When you open this file, you should see the exact same code as the plugin code in your tool workspace (~/your_tool_workspace/backend/ayx_plugins/<tool_name>.py). If you make some changes to your code in the workspace, all updates should propagate to this file that's also in the .yxi that you build. This step is used to confirm that the .yxi is built with the correct code and dependencies. You can also search for dependencies in the site-packages folder of the archive to verify that the other plugin dependencies are correctly built.

Reply:
yeah this sounds better, thank you!